src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/stop-interaction.mdx

```diff
@@ -30,21 +30,21 @@ freshnessValidatedDate: 2023-07-21
         <TabsBarItem id="cordova">
             Cordova
         </TabsBarItem>
-       <TabsBarItem id="maui">
-            .NET MAUI
-        </TabsBarItem>
         <TabsBarItem id="flutter">
             Flutter
         </TabsBarItem>
+        <TabsBarItem id="maui">
```

Comment:
@austin-schaefer I think there's something wrong with the IDs. When I click on Flutter, I am taken to .NET MAUI, and when I click on Unity, I'm taken to Xamarin.